.devcontainer/images/Dockerfile.base

```
@@ -0,0 +1,145 @@
+    curl -fsSL "https://github.com/mikefarah/yq/releases/latest/download/yq_linux_${ARCH}" -o /usr/bin/yq && \
+    chmod +x /usr/bin/yq
+
+# =============================================================================
+# User Setup (directories + Oh My Zsh + shell config)
+# =============================================================================
+USER vscode
+WORKDIR /home/vscode
+
+COPY --chown=vscode:vscode .p10k.zsh /home/vscode/.p10k.zsh
+
+RUN mkdir -p .cache .config .local/bin .local/share .zsh_history_dir \
```

Hadolint (2.14.0)

[warning] In POSIX sh, brace expansion is undefined.

(SC3009)